Hast.Abstractions/Hast.Vitis.Abstractions/Services/VitisHardwareImplementationComposerBuildProvider.cs

```diff
@@ -577,13 +583,18 @@ private async Task EnsureDeviceReadyAsync(VitisBuildConfiguration buildConfigura
 
         if (promptBeforeBuild)
         {
+            // This is not logging but a prompt in select cases.
+#pragma warning disable S106 // Standard outputs should not be used directly to log anything
+#pragma warning disable S2228 // Console logging should not be used

```

Comment:
Is it necessary to enable both warnings? Seems redundant.

Reply:
Indeed, and [S106 is the rule that's the current one](https://github.com/SonarSource/sonar-dotnet/issues/1812).